apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx

```diff
@@ -160,17 +160,17 @@ const SideNav = () => {
     <div className="flex relative">
       {/* // main menu */}
       <NavigationMenuHome />
-      <NavigationMenuGuideList id={'gettingstarted'} />
-      <NavigationMenuGuideList id={'database'} />
-      <NavigationMenuGuideList id={'auth'} />
-      <NavigationMenuGuideList id={'functions'} />
-      <NavigationMenuGuideList id={'realtime'} />
-      <NavigationMenuGuideList id={'storage'} />
-      <NavigationMenuGuideList id={'platform'} />
-      <NavigationMenuGuideList id={'resources'} />
-      <NavigationMenuGuideList id={'integrations'} />
-      <NavigationMenuGuideList id={'self_hosting'} />
-      <NavigationMenuGuideList id={'reference'} />
+      <NavigationMenuGuideList id={'gettingstarted'} context={context} />
```

Comment:
We need to pass something in here ("side" or "mobile" as we're rendering all these menus twice. 

We need a way to differentiate the keys on the mapped items.

Reply:
fixed in another PR

Reply:
https://github.com/supabase/supabase/pull/10930

just fixing mobile stuff now.
fixed the duplicate menu issue locally, about to push